src/main/peloton/peloton.cpp

```diff
@@ -83,20 +79,41 @@ int main(int argc, char *argv[]) {
   try {
     // Print settings
     if (peloton::settings::SettingsManager::GetBool(
-      peloton::settings::SettingId::display_settings)) {
+            peloton::settings::SettingId::display_settings)) {
       auto &settings = peloton::settings::SettingsManager::GetInstance();
       settings.ShowInfo();
     }
   } catch (peloton::SettingsException &exception) {
-    peloton::LOG_ERROR("Cannot load settings. Failed with %s\n", exception.GetMessage().c_str());
-    return EXIT_FAILURE; // TODO: Use an enum with exit error codes
+    peloton::LOG_ERROR("Cannot load settings. Failed with %s\n",
+                       exception.GetMessage().c_str());
+    return EXIT_FAILURE;  // TODO: Use an enum with exit error codes
   }
 
   int exit_code = 0;
   if (peloton::settings::SettingsManager::GetBool(
-      peloton::settings::SettingId::brain))
-    exit_code =  RunPelotonBrain();
+          peloton::settings::SettingId::brain))
+    exit_code = RunPelotonBrain();
   else
     exit_code = RunPelotonServer();
+
+  // TODO[Siva]: Remove this from the final PR. Uncomment this to run brain
+  // and server in the same process for testing. This is a temporary to way to
+  // run both peloton server and the brain together to test the index suggestion
+  // at the brain end without catalog replication between the server and the
+  // brain
+  // peloton::settings::SettingsManager::SetBool(
```

Comment:
We will remove this in the final PR. We currently have it because we have different things to test for performance and this is a quick way to test the 'online' index suggestion (see IndexSelectionJob)